README.md

```diff
@@ -205,6 +214,41 @@ interface GatewayConfig {
 }
 ```
 
+### `proxyHeaders`
+
+`proxyHeaders` is an optional method that allows setting headers for requests at the entire `gateway` level.
+It is recommended to use this method when gateway methods are defined not only locally but also imported from external libraries.
```

Comment:
I would not use a recommendation style to describe the work of `proxyHeaders`. Let's describe that there are 2 ways to set up `proxyHeaders`: on gateway config level and on action config level

Reply:
there is sign about second way to use

https://github.com/gravity-ui/gateway/pull/130/files